manifests/node-exporter.yml

```diff
@@ -29,5 +29,5 @@ stemcells:
 releases:
 - name: node-exporter
   version: 5.5.0
-  url: https://github.com/bosh-prometheus/node-exporter-boshrelease/releases/download/v5.5.0/node-exporter-5.5.0.tgz
+  url: https://github.com/cloudfoundry/node-exporter-boshrelease/releases/download/v5.5.0/node-exporter-5.5.0.tgz
```

Comment:
@benjaminguttmann-avtq, maybe it's better to wait until the next release is produced to change the release url in the manifest, what do you think?